features/admin.users.v1/api/users.ts

```diff
@@ -470,3 +470,61 @@ export const terminateAllUserSessions = (userId: string): Promise<AxiosResponse>
                 error.config);
         });
 };
+
+/**
+ * Interface for the resend code request payload.
+ */
+export interface ResendCodeRequest {
+    user: {
+        username: string;
+        realm: string;
+    };
+    properties: Array<{
+        key: string;
+        value: string;
+    }>;
+}
+
+/**
+ * Resends the verification code for the for recovery-related scenarios.
+ *
+ * @param data - The request payload containing user information and properties.
+ * @returns A promise containing the response.
+ * @throws `IdentityAppsApiException` if the request fails or if the response status is not as expected.
+ */
+export const resendCode = (data: ResendCodeRequest): Promise<any> => {
+
+    const requestConfig: RequestConfigInterface = {
+        data,
+        headers: {
+            "Access-Control-Allow-Origin": store.getState().config.deployment.clientHost,
```

Comment:
Access-Control-* headers are to be sent by the server, not by client.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Access-Control-Allow-Origin

```suggestion
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/fb3422572ea4918d0817c4d3802d2137fa0dd495